app/lib/features/super_invites/widgets/invite_list_item.dart

```diff
@@ -36,12 +38,16 @@ class InviteListItem extends StatelessWidget {
             style: Theme.of(context).textTheme.labelMedium,
           ),
           onTap: () {
-            context.pushNamed(
-              Routes.createSuperInvite.name,
-              extra: inviteToken,
-            );
+            if (onSelectInviteCode != null && inviteToken != null) {
+              onSelectInviteCode!(inviteToken!);
```

Comment:
```suggestion
            if (onSelectInviteCode != null) {
              onSelectInviteCode!(inviteToken);
```

final Function(SuperInviteToken)? onSelectInviteCode returns nullable data no need to check for null condition here.

Reply:
Actually `SuperInviteToken` is not nullable callback function is nullable, and the `invitetoken != null` condition is already handled in the onTap method, so it will not cause any issues. I need to put this condition because I have to pass `invitetoken` for super invite code item to  `onClick` method and also it's helpful in test cases.